test/jpypetest/test_jclass.py

```diff
@@ -204,3 +204,21 @@ def testInnerClass(self):
         test = JClass("jpype.types.InnerTest")()
         test.test()
 
+    def testLookupGeneric(self):
```

Comment:
Is a not-found class mapped to None? Or what is the desired outcome?

Reply:
Not found should raise if I remember correctly.